highest-scoring-word/solution.js

```diff
@@ -0,0 +1,5 @@
+function high(x){
+  let word = x.split(' ')
+  .map(x=>x.split('').reduce((a,b)=>a+b.charCodeAt(0)-96,0));
```

Comment:
forEach() would be more appropriate to use than .map() for this kata.
It works, but like we said yesterday, this kata's challenge was not to transform data but to evaluate data.

forEach(), like .map() and .reduce(), is declarative (not iterative), even though it's iterating behind the scenes, so it'd be good practice all around in this kata.